bindata/etcd/pod.yaml

```diff
@@ -103,6 +103,34 @@ ${COMPUTED_ENV_VARS}
       - mountPath: /var/lib/etcd/
         name: data-dir
     env:
+${COMPUTED_ENV_VARS}
+  - name: etcd-healthz
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -euo pipefail
+        cluster-etcd-operator healthz-proxy \
+          --listen-port=10100 \
+          --listen-cert=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-serving-metrics-NODE_NAME.crt \
+          --listen-key=/etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-serving-metrics-NODE_NAME.key \
+          --client-cert=${ETCDCTL_CERT} \
+          --client-key=${ETCDCTL_KEY} \
+          --client-cacert=${ETCDCTL_CACERT}
+    ports:
+      - containerPort: 10100
```

Comment:
yeah the port was just chosen out of the air so that does need to be changed.

https://github.com/openshift/cluster-etcd-operator/pull/565/files#diff-fad938af40231086decc9dad2ad6a274c6ba5d4a4cdc8696d246d84b5a1770beR34